src/types/captcha.ts

```diff
@@ -96,19 +92,12 @@ export interface CaptchaSolutionCommitment {
     completed_at: u64,
 }
 
-type CaptchaSolutionBase = {
+export interface CaptchaSolution {
     captchaId: string;
     salt: string;
-};
-
-export interface CaptchaSolution extends CaptchaSolutionBase {
     solution: HashedSolution[];
 }
 
-export interface CaptchaSolutionRaw extends CaptchaSolutionBase {
```

Comment:
Why does this go away? Isn't it needed for adding a dataset in which the solutions are arrays of numbers?

Reply:
I believe you mean `CaptchaWithoutIdRaw`. this is not used anymore

Reply:
Great, thanks for clarifying.